config/routes.rb

```diff
@@ -5,7 +5,11 @@
  
   namespace 'api' do
     namespace 'v1' do
-      resources :targets
+      resources :targets do
```

Comment:
remember here to specify only the resources that you want so you don't publish endpoints that you don't have